diff --git a/node_modules/@api-ts/openapi-generator/dist/src/openapi.js b/node_modules/@api-ts/openapi-generator/dist/src/openapi.js
index 849524e..9629aca 100644
--- a/node_modules/@api-ts/openapi-generator/dist/src/openapi.js
+++ b/node_modules/@api-ts/openapi-generator/dist/src/openapi.js
@@ -319,7 +319,7 @@ function routeToOpenAPI(route) {
             ...(jsdoc.description !== undefined ? { description: jsdoc.description } : {}),
             ...(operationId !== undefined ? { operationId } : {}),
             ...(tag !== '' ? { tags: [tag] } : {}),
-            ...(isInternal ? { 'x-internal': true } : {}),
+            ...{ 'x-internal': isInternal },
             ...(isUnstable ? { 'x-unstable': true } : {}),
             ...(Object.keys(unknownTagsObject).length > 0
                 ? { 'x-unknown-tags': unknownTagsObject }
