diff --git a/node_modules/@polkadot/x-fetch/cjs/node.js b/node_modules/@polkadot/x-fetch/cjs/node.js
index 2917a9d..dc41a1a 100644
--- a/node_modules/@polkadot/x-fetch/cjs/node.js
+++ b/node_modules/@polkadot/x-fetch/cjs/node.js
@@ -4,16 +4,25 @@ exports.fetch = exports.packageInfo = void 0;
 const x_global_1 = require("@polkadot/x-global");
 var packageInfo_js_1 = require("./packageInfo.js");
 Object.defineProperty(exports, "packageInfo", { enumerable: true, get: function () { return packageInfo_js_1.packageInfo; } });
-const importFetch = import('node-fetch').catch(() => null);
+// COMMENTED CODE CAUSES: "ReferenceError: __import__ is not defined"
+// const importFetch = import('node-fetch').catch(() => null); 
 let modFn = null;
+
 async function nodeFetch(...args) {
     if (!modFn) {
-        const mod = await importFetch;
-        if (!mod?.default) {
-            throw new Error('Unable to import node-fetch in this environment');
+        try {
+            // Dynamically import node-fetch
+            const mod = await (new Function("return import('node-fetch')"))();
+            if (!mod?.default) {
+                throw new Error('Unable to import node-fetch in this environment');
+            }
+            modFn = mod.default;
+        } catch (error) {
+            console.error("Error importing node-fetch:", error);
+            throw error;
         }
-        modFn = mod.default;
     }
     return modFn(...args);
 }
+
 exports.fetch = (0, x_global_1.extractGlobal)('fetch', nodeFetch);
\ No newline at end of file
