diff --git a/node_modules/nise/lib/event/custom-event.js b/node_modules/nise/lib/event/custom-event.js
index c4858d4..a55fccf 100644
--- a/node_modules/nise/lib/event/custom-event.js
+++ b/node_modules/nise/lib/event/custom-event.js
@@ -9,6 +9,10 @@ function CustomEvent(type, customData, target) {
 
 CustomEvent.prototype = new Event();
 
-CustomEvent.prototype.constructor = CustomEvent;
-
+Object.defineProperty(CustomEvent.prototype, 'constructor', {
+    value: CustomEvent,
+    writable: false,
+    enumerable: false,
+    configurable: false
+});
 module.exports = CustomEvent;
diff --git a/node_modules/nise/lib/event/progress-event.js b/node_modules/nise/lib/event/progress-event.js
index dba834c..ad0e77e 100644
--- a/node_modules/nise/lib/event/progress-event.js
+++ b/node_modules/nise/lib/event/progress-event.js
@@ -11,6 +11,10 @@ function ProgressEvent(type, progressEventRaw, target) {
 
 ProgressEvent.prototype = new Event();
 
-ProgressEvent.prototype.constructor = ProgressEvent;
-
+Object.defineProperty(ProgressEvent.prototype, 'constructor', {
+    value: ProgressEvent,
+    writable: false,
+    enumerable: false,
+    configurable: false
+});
 module.exports = ProgressEvent;
