commit d8cbb7e8241a9d7a735806a39388b1989c48e21a
Author: Otto Allmendinger <otto@bitgo.com>
Date:   Thu Feb 13 15:22:23 2025 +0100

    Apply changes to make limited codebase work

diff --git a/btc-staking-ts/src/index.ts b/btc-staking-ts/src/index.ts
index b48f5ca05..1dcec4043 100644
--- a/btc-staking-ts/src/index.ts
+++ b/btc-staking-ts/src/index.ts
@@ -13,5 +13,4 @@ export * from "./utils/utxo/getScriptType";
 export {
   getBabylonParamByBtcHeight,
   getBabylonParamByVersion,
-} from "./utils/staking/param";
-export * from "./staking/manager";
\ No newline at end of file
+} from "./utils/staking/param";
\ No newline at end of file
diff --git a/btc-staking-ts/src/staking/observable/index.ts b/btc-staking-ts/src/staking/observable/index.ts
index 9d69a9319..ebf4db36c 100644
--- a/btc-staking-ts/src/staking/observable/index.ts
+++ b/btc-staking-ts/src/staking/observable/index.ts
@@ -6,7 +6,7 @@ import { isTaproot } from "../../utils/btc";
 import { toBuffers, validateStakingTxInputData } from "../../utils/staking";
 import { TransactionResult } from "../../types/transaction";
 import { ObservableStakingScriptData, ObservableStakingScripts } from "./observableStakingScript";
-import { StakerInfo, Staking } from "..";
+import { StakerInfo, Staking } from "../index";
 import { networks, Psbt, Transaction } from "bitcoinjs-lib";
 import { stakingPsbt } from "../psbt";
 export * from "./observableStakingScript";
diff --git a/btc-staking-ts/src/utils/btc.ts b/btc-staking-ts/src/utils/btc.ts
index 2f9123497..75c9bf9ab 100644
--- a/btc-staking-ts/src/utils/btc.ts
+++ b/btc-staking-ts/src/utils/btc.ts
@@ -1,12 +1,7 @@
-import * as ecc from "@bitcoin-js/tiny-secp256k1-asmjs";
-import { initEccLib, address, networks } from "bitcoinjs-lib";
+import { ecc } from '@bitgo/utxo-lib';
+import { address, networks } from "bitcoinjs-lib";
 import { NO_COORD_PK_BYTE_LENGTH } from "../constants/keys";
 
-// Initialize elliptic curve library
-export const initBTCCurve = () => {
-  initEccLib(ecc);
-}
-
 /**
  * Check whether the given address is a valid Bitcoin address.
  *
